### DISCUSSION TITLE
Cutom sort function/ data accessor

### DISCUSSION THREAD

**Discussion by enejm** [ng-matero/extensions#135] [Q&A] [answered]
Hi. 

First of all thank you for fantastic library. 🙏

I am having question regarding Data Grid. Is it possible to add custom sort function or set up data accessor like with material table?
I am having nested data. With formatter I can display data, but it does't work for sorting. 

```
columns: MtxGridColumn[] = [
    {
      header: 'Name',
      field: 'user',
      formatter: (data: Report) => `${data.user.firstname + ' ' + data.user.lastname}`,
      sortable: true,
    },
  ];
```

**Comment by nzbin** [comment, marked as answer]
```html
<mtx-grid #grid></mtx-grid>
```

```ts
@ViewChild('grid') grid: MtxGridComponent;

// this.grid.table => MatTable
// this.grid.sort => MatSort
// this.grid.dataSource => MatTableDataSource
```

I haven't used the `sortingDataAccessor` yet. You can try it and give me some advice.
